src/test/java/teammates/ui/webapi/GetNotificationActionTest.java

```diff
@@ -0,0 +1,83 @@
+package teammates.ui.webapi;
+
+import org.testng.annotations.BeforeMethod;
+import org.testng.annotations.Test;
+
+import teammates.common.datatransfer.attributes.NotificationAttributes;
+import teammates.common.util.Const;
+import teammates.ui.output.NotificationData;
+
+/**
+ * SUT: {@link GetNotificationAction}.
+ */
+public class GetNotificationActionTest extends BaseActionTest<GetNotificationAction> {
+
+    @Override
+    String getActionUri() {
+        return Const.ResourceURIs.NOTIFICATION;
+    }
+
+    @Override
+    String getRequestMethod() {
+        return GET;
+    }
+
+    @BeforeMethod
+    public void baseClassSetup() {
+        loginAsAdmin();
+    }
+
+    @Test
+    @Override
+    protected void testAccessControl() {
+        verifyOnlyAdminCanAccess();
+    }
+
+    @Test
```

Comment:
This test annotation can be omitted since no test method are executed here

Reply:
Okay thank you!